### ISSUE TITLE
How to resolve "may require use of '--include-plugin-directory' warnings?

### ISSUE THREAD

**Issue by mcondarelli** [Nuitka/Nuitka#214]
$ python3 -m nuitka --version
0.6.1
Python: 3.7.2 (default, Jan  3 2019, 02:55:40) 
Executable: /usr/bin/python3
OS: Linux
Arch: x86_64

Installed via deb (on debian/sid).
No virtual environment (using system installation).

I am trying to compile locally a rather complex program using a lot of different libraries, in preparation for attempt to cross-compilation (see #43).

This is the compilation output:
```
$ python3 -m nuitka --follow-imports --standalone --plugin-enable=pylint-warnings stream.py
Nuitka:WARNING:Unresolved '__import__' call at '/usr/lib/python3/dist-packages/jinja2/loaders.py:472' may require use of '--include-plugin-directory' or '--include-plugin-files'.
Nuitka:WARNING:Unresolved '__import__' call at '/usr/lib/python3/dist-packages/jinja2/utils.py:142' may require use of '--include-plugin-directory' or '--include-plugin-files'.
Nuitka:WARNING:Unresolved '__import__' call at '/usr/lib/python3/dist-packages/jinja2/utils.py:143' may require use of '--include-plugin-directory' or '--include-plugin-files'.
Nuitka:WARNING:Unresolved '__import__' call at '/usr/lib/python3/dist-packages/jsonpickle/backend.py:104' may require use of '--include-plugin-directory' or '--include-plugin-files'.
Nuitka:WARNING:Unresolved '__import__' call at '/usr/lib/python3/dist-packages/jsonpickle/unpickler.py:548' may require use of '--include-plugin-directory' or '--include-plugin-files'.
Nuitka:WARNING:Unresolved '__import__' call at '/usr/lib/python3/dist-packages/jsonpickle/unpickler.py:611' may require use of '--include-plugin-directory' or '--include-plugin-files'.
Nuitka:WARNING:Unresolved '__import__' call at '/usr/lib/python3/dist-packages/jsonpickle/util.py:323' may require use of '--include-plugin-directory' or '--include-plugin-files'.
Nuitka:WARNING:Unresolved '__import__' call at '/usr/lib/python3/dist-packages/requests/packages.py:7' may require use of '--include-plugin-directory' or '--include-plugin-files'.
```
I had a look to the offending locations and they seem to reference library packages.
What should I do to "resolve" the warnings?
I found no documentation, can someone point me to the right place, please?

**Comment by kayhayen** [comment]
There is no way but through the plugin API to disable these warnings.

My idea was to white list all packages from standard libraries that do this, and are known to still work, despite not doing anything about it, which never got traction.

Most often the place that does the `__import__` call is rather remote or optionally called only. Might just give up on those warnings as well, keeping them to the plugin API level only for informational purposes.

Yours,
Kay

**Comment by kayhayen** [comment]
Yes, I removed these warnings already, for similar reports. Nobody ever notices anything from them, most of the time it's pretty normal to not find modules, e.g. due to Python2/3 support, and I got fed up with white listing, it doesn't scale. On the other hand, users never looked at those warnings.

Of course, since I remove them, I got complaints too.

The intended solution is a report, that tabulates things and can be looked at and checked for changes too. But this work is not happening yet.
